acoular/spectra.py

```diff
@@ -187,7 +187,7 @@ class PowerSpectra(BaseSpectra):
     _ind_low = Int(1, desc='index of lowest frequency line')
 
     # Shadow trait, should not be set directly, for internal use.
-    _ind_high = Trait(-1, (Int, None), desc='index of highest frequency line')
+    _ind_high = Union(Int(-1), None, desc='index of highest frequency line')
```

Comment:
is the default value missing here (`-1`)?

Reply:
no, its not, see `Int(-1)`
First argument of `Union` is the default.

Reply:
Ah sorry, missed that.